### ISSUE TITLE
accept host and port as envvars

### ISSUE THREAD

**Issue by ashleygwilliams** [steveklabnik/simple-server#25]

**Comment by steveklabnik** [comment]
This is basically upgrading the examples, right? This is already supported by the library.

**Comment by ashleygwilliams** [comment]
ah i was thinking we could bake it in, but that's probably not necessary (like include dotenv and stuff). i'm gonna leave this open cuz it might be nice, but it's not urgent.

we might add defaults? 

i was thinking something like host and port are optional and they default to:

- `0.0.0.0 || process.env.HOST`
- `7878 || process.env.POST`

**Comment by steveklabnik** [comment]
So, the trick there is that Rust doesn't have default arguments, so you'd have to make a second method that had no arguments and used the vars by default.

**Comment by ashleygwilliams** [comment]
ah yeah this seems like a nice idea

**Comment by jonashagstedt** [comment]
I would argue that using 0.0.0.0 as default is not as safe as using 127.0.0.1.
This means that the web server is instantly accessible on your network and if you do happen to have port forwarding on it would become exposed online, so defaulting to 127.0.0.1 is safer.

**Comment by steveklabnik** [comment]
@jonashagstedt yeah I always forget which one is the good one and which one isn't; we should default to the good one.
